publications/ro2019.md

```diff
@@ -28,7 +28,9 @@ To help the end user of the dataset, our package additionally cleans and standar
 
 2) Enforcing that every file path found exists prior to attempting distribution. (Commonly, when collaboratively working on shared datasets, files will move to a new location during the process of development.)
 
-Learning from the extensive work in linked data, retention of file associations is an incredibly valuable tool for not only navigation between objects but also the relationships between those objects[1]. To apply these linked data concepts in our own application, if a manifest has multiple columns that contain file paths to resources, our application additionally constructs a lookup table for those shared file associations and places the relevant portion of the lookup table in each file's metadata. An example of this behavior can be seen when an imaging manifest has a column for full size images and a column for thumbnail images. For each row of the manifest, the application makes an entry in the Quilt package that pairs the full size image to the smaller thumbnail image by storing the read path to the other in the metadata for each member of the pair.
+Learning from the extensive work in linked data, retention of file associations [why not just say provenance?] is an incredibly valuable tool for not only navigation between objects but also the relationships between those objects[1]. To apply these linked data concepts in our own application, if a manifest has multiple columns that contain file paths to resources, our application additionally constructs a lookup table for those shared file associations and places the relevant portion of the lookup table in each file's metadata. An example of this behavior can be seen when an imaging manifest has a column for full size images and a column for thumbnail images. For each row of the manifest, the application makes an entry in the Quilt package that pairs the full size image to the smaller thumbnail image by storing the read path to the other in the metadata for each member of the pair.
```

Comment:
Provenance doesn't necessarily just mean history of HOW a thing was produced, but also what types of things are produced... provenance can be used to describe the set of relationships that exist when a data object is transformed, derived, subsetted, etc. 

I understood your example as a derivative object (e.g. a thumbnail is derived from a full size image) ... what I meant is that you are typing relationships between products based on their provenance (this thing came from that thing) ...

The interesting aspect of this is not that a link exists, but that the link is a particular kind of link... a predicate that established a relationship between a subject and an object... And in this case the predicates have to do with relationships between different objects of the same dataset... right? 
- Subject->predicate->object 
- thumbnail->is_derived_from->image
- image->has_derivations->thumbnail

Reply:
Ahhhhh I get it. I think it would be incredibly neat if this software had the ability to annotate something like that and may in fact be added to future work. But currently no "annotations" get added to the metadata about the linkage itself.

In this case I am defining linkage annotation as that middle piece:

* Subject -> *annotation:* predicate -> object
* thumbnail -> *annotation:* is_derived_from -> image
* image -> *annotation:* has_derivations -> thumbnail

Reply:
In my system it really is just storing in the full image's metadata:
```
"thumbnail": "/path/to/thumbnail.png"
```

and storing in the thumbnail's metadata:
```
"full_image": "/path/to/full_image.tiff"
```